src/System.Management.Automation/resources/DebuggerStrings.resx

```diff
@@ -246,4 +246,25 @@ The current session does not support debugging; operation will continue.
   <data name="InvalidDetachCommand" xml:space="preserve">
     <value>The debugger detach command is not applicable.  The detach command only applies when debugging jobs and runspaces with the Debug-Job or Debug-Runspace cmdlets.</value>
   </data>
-</root>
+  <data name="BreakInstruction" xml:space="preserve">
+    <value>To break into the debugger when a command or script is running, press 'Ctrl+Break'.</value>
+  </data>
+  <data name="DebuggingJob" xml:space="preserve">
+    <value>Debugging Job: {0}</value>
+  </data>
+  <data name="DetachInstruction" xml:space="preserve">
+    <value>To detach the debugger, invoke 'detach' from the debugger prompt, or press 'Ctrl+C' when running.</value>
+  </data>
+  <data name="ExitInstruction" xml:space="preserve">
+    <value>To exit the debugger, press 'Ctrl+C'.</value>
+  </data>
+  <data name="ScriptCompletedMessage" xml:space="preserve">
+    <value>Command or script completed.</value>
+  </data>
+  <data name="DebuggingNoDebugger" xml:space="preserve">
+    <value>Cannot debug because the host has no debugger. Try debugging inside the PowerShell console or with Visual Studio Code, both of which have built-in debuggers.</value>
+  </data>
+  <data name="DebuggingNoHost" xml:space="preserve">
+    <value>Cannot debug because there is no host or host UI. The debugger requires a host and host UI for debugging.</value>
+  </data>
+</root>
```

Comment:
Please fix all resx files in the PR - I see EOL here and extra spaces.

Reply:
I didn't make any of those changes. I don't edit these by hand (with one exception -- there was a merge conflict, and I resolved that by hand, but it didn't impact the newline nor the spaces). Further, the other resx file does not have an extra EOL at the end of the file either, but it's not called out because the last entry didn't change. I think these come from tooling (I use Visual Studio 2019 latest update for resource string changes), so I'm not sure what should be done here.

Reply:
GitHub diff shows that new spaces was added in many places. Perhaps VS 2019 did accidentally this. But in any case we need revert this.

Reply:
No need to apologize. I'll keep an eye on this one with other PRs where I change resource strings because ultimately, if tooling changes to the point where we need to fight against it, we may need to take on such changes in an independent PR. I'll only suggest that if this becomes an ongoing, persistent issue to deal with when making resource file changes.

Reply:
Also, these unintended whitespace and https->http changes have been reverted.